src/components/slices/intro-description.module.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../base/module';
+
+.introBox {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  padding: 110px 0;
+  background-color: $color-dove;
+  position: relative;
+  margin-bottom: 10px;
+}
+
+.palm::before {
+  content: '';
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: -10px;
+  z-index: 1;
+  width: 360px;
+  height: 190px;
+  background-color: $color-sage;
+  mask: url('../../images/palm.svg') no-repeat 100% 100%;
+  mask-size: contain;
+  transform: rotate(-10deg) translateX(-50%) translateY(10%);
+  @media (min-width: $desktop) {
```

Comment:
I would add a tablet size of palm personally, it's going small quite early and I think it would look nicer to have a third size level.